src/app/rosetta/README.md

```diff
@@ -1,97 +1,117 @@
 # Rosetta
 
-Implementation of the [Rosetta API](https://www.rosetta-api.org/) for Mina.
-
-## How to build your own docker image
-
-Checkout the "master" branch of the mina repository, ensure your Docker configuration has a large amount of RAM (at least 12GB, recommended 16GB) and then run the following:
-
-### Release v1.3.1.2 (Mainnet and Devnet)
-
-```
-cat dockerfiles/stages/1-build-deps \
-dockerfiles/stages/2-opam-deps \
-    dockerfiles/stages/3-builder \
-    dockerfiles/stages/4-production \
-    | docker build -t mina-rosetta-ubuntu:v1.3.1.2 \
-        --build-arg "DUNE_PROFILE=mainnet" \
-        --build-arg "MINA_BRANCH=master" -
-```
-
-This creates an image (mina-rosetta-ubuntu:v1.3.1.2) based on Ubuntu
-20.04 and includes the most recent release of the mina daemon along
-with mina-archive and mina-rosetta. Note the `DUNE_PROFILE` argument,
-which should take a different value depending on which network you
-want to connect to. Complete list of predefined `dune` profiles can be
-found in `src/config` directory in the main repository.
-
-Alternatively, you could use the official image `minaprotocol/mina-rosetta:1.3.1.2-25388a0-bullseye` which is built in exactly this way by buildkite CI/CD.
-
-### Release v2.0.0 (Berkeley)
-
-```
-cat dockerfiles/stages/1-build-deps \
-dockerfiles/stages/2-opam-deps \
-    dockerfiles/stages/3-builder \
-    dockerfiles/stages/4-production \
-    | docker build -t mina-rosetta-ubuntu:v2.0.0 \
-        --build-arg "DUNE_PROFILE=devnet" \
-        --build-arg "MINA_BRANCH=develop" -
+Implementation of the [Rosetta API](https://docs.cloud.coinbase.com/rosetta/docs/welcome) for Mina.
+
+## How to build your Docker image
+
+Checkout the branch of the Mina repository matching the network you wish to use,
+ensure your Docker configuration has a large amount of RAM (at least 12GB,
+recommended 16GB) and then run the following:
+
+```bash
+cat dockerfiles/Dockerfile-mina-rosetta \
+    | docker build -t mina-rosetta:v2.0.0 \
+        --build-arg "network=berkeley" \
```

Comment:
There's a chance people will copy-paste this command into their terminals. Most often they'll probably want to build for mainnet, won't they? I'd include `network=mainnet` here rather than `berkeley`, which will become irrelevant after the hard fork.

Reply:
I agree, but we still don't have releases tagged for `mainnet` using the `berkeley` branch, therefore this would render the command invalid as of now. Let's update it later once it is released.